### PR TITLE
Check in openipmi qemu deb package builder scripts.

### PR DESCRIPTION
**Pull request by JuneZhou19**
@xiar @fub2 @turtle-fly @PaynePei

### CODE REVIEW

File: deb_builder/qemu-package.sh [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+JOBS=32
+
+[ -n "$CHECKINSTALL" ] || die "Please install checkinstall."
+
+pkgrelease=1
+src=""
+while [ $# -gt 0 ]; do
+    option=$1
+    case $option in
+    --package-type) type=$2;shift ;;
```
Comment (fub2) [inline deb_builder/qemu-package.sh]: Type isn't used; what is it for?

File: deb_builder/openipmi-package.sh [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+JOBS=32
+
+[ -n "$CHECKINSTALL" ] || die "Please install checkinstall."
+
+pkgrelease=1
+src=""
+while [ $# -gt 0 ]; do
+    option=$1
+    case $option in
+    --package-type) type=$2;shift ;;
```
Comment (fub2) [inline deb_builder/openipmi-package.sh]: Type isn't used; what is it for?

File: deb_builder/qemu-package.sh (hunk 1 above)
Comment (JuneZhou19) [inline deb_builder/qemu-package.sh]: Yes, it's not used. Just removed it.

**Comment by JuneZhou19** [comment]
@xiar @fub2 @MarkMa001 @turtle-fly @PaynePei @XiaowenJiang

**Comment by fub2** [comment]
+1

**Comment by turtle-fly** [comment]
+1

**Pull request merged**